src/VisualStudio/CSharp/Impl/Options/CSharpSettingsManagerOptionSerializer.cs

```diff
@@ -65,6 +65,7 @@ public CSharpSettingsManagerOptionSerializer(VisualStudioWorkspaceImpl workspace
         private const string Style_UseImplicitTypeWherePossible = nameof(AutomationObject.Style_UseImplicitTypeWherePossible);
         private const string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration = nameof(AutomationObject.Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration);
         private const string Style_PreferIntrinsicPredefinedTypeKeywordInMemberAccess = nameof(AutomationObject.Style_PreferIntrinsicPredefinedTypeKeywordInMemberAccess);
+        private const string Style_UseBracesWherePossible = nameof(AutomationObject.Style_UseBracesWherePossible);
 
```

Comment:
This is an odd name. "Use braces where possible".  Aren't braces always possible?  Maybe this should be the reverse "Remove braces when not necessary."